app/code/Magento/Downloadable/Test/Unit/Model/Sample/DeleteHandlerTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Downloadable\Test\Unit\Model\Sample;
+
+use Magento\Catalog\Api\Data\ProductInterface;
+use Magento\Catalog\Model\Product;
+use Magento\Downloadable\Api\Data\SampleInterface;
+use Magento\Downloadable\Api\SampleRepositoryInterface as SampleRepository;
+use Magento\Downloadable\Model\Product\Type;
+use Magento\Downloadable\Model\Sample\DeleteHandler;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit Test for \Magento\Downloadable\Model\Sample\DeleteHandler
+ */
+class DeleteHandlerTest extends TestCase
+{
+    const STUB_PRODUCT_TYPE = 'simple';
```

Comment:
Just a recommendation for the future to declare constant scope as `private`. It will reduce the chance of "temptation" to make another dependency on the current test in another test.

Reply:
Thanks for the note, will keep it in mind!